src/containers/ProgressTrackerContainer.jsx

```diff
@@ -56,56 +57,72 @@ class ProgressTrackerContainer extends Component {
   }
 
   render() {
-    const { genreqs, majorreqs, isGenReq, isMajorReq, isMathsci, isAhse, isEngr,
+    const { genreqs,
+      majorreqs,
+      isGenReq,
+      isMajorReq,
+      isMathsci,
+      isAhse,
+      isEngr,
       mathTotal,
       engrTotal,
       sciTotal,
       ahseTotal,
-      mathCourses, engrCourses, sciCourses, ahseCourses } = this.props;
+      engrPercent,
+      ahsePercent,
+      mathSciPercent,
+      genreqsPercent,
+      majorreqsPercent,
+      mathCourses,
+      engrCourses,
+      sciCourses,
+      ahseCourses,
+    } = this.props;
+
     if (isGenReq) {
       const genreqItems = genreqs.map(req => <li>{req}</li>);
       return (
-        <div>
+        <div className={styles.trackerComponent}>
           <a onClick={this.backReq} href=""> ← back</a>
-          <p>{genreqs.length}/16 courses</p>
-          <ul>{genreqItems}</ul>
+          <p className={styles.progressRatio}>{genreqs.length}/16 courses</p>
+          <ul className={styles.classList}>{genreqItems}</ul>
         </div>
       );
     } else if (isMajorReq) {
       const majorreqItems = majorreqs.map(req => <li>{req}</li>);
       return (
-        <div>
+        <div className={styles.trackerComponent}>
           <a onClick={this.backReq} href=""> ← back</a>
-          <p>{majorreqs.length}/7 courses</p>
-          <ul>{majorreqItems}</ul>
+          <p className={styles.progressRatio}>{majorreqs.length}/7 courses</p>
+          <ul className={styles.classList}>{majorreqItems}</ul>
```

Comment:
Can we have some way of knowing that the courses listed below the back link are the completed ones?